src/Adapter/Carrier/Repository/CarrierRepository.php

```diff
@@ -102,16 +102,16 @@ public function updateInNewVersion(CarrierId $carrierId, Carrier $carrier): Carr
         if ($carrier->delay) {
             $newCarrier->delay = $carrier->delay;
         }
-        if ($carrier->max_width) {
+        if (null !== $carrier->max_width) {
```

Comment:
As we discusses this can probably be simplified in the next optimization PR, but ok for now